### PR TITLE
complete overhaul of the loadout system for waster/tribal/outlaw

### PR DESCRIPTION
**Pull request by lolman360**
tldr: pointbuy for waster/tribal/outlaw, outlaws get slightly more points because fuck you

## About The Pull Request

makes loadout point-buy using uplinks. removes loadouts for wastelander, tribal and outlaw since they get this system instead.

uplink disappears when you run out of tc, and is stuck to your hand otherwise, discouraging saving the uplink for an asspull later

if the possibility becomes 'funny' instead of 'hyperabuseable' i might change it

## Why It's Good For The Game
current loadout system is kind of restrictive especially for these very freeform roles

it can stick around for major factions IMO, but minor ones and possibly BoS might be moved to this system later down the line since 1. ostensibly they'd be less rigid in the gear they use and 2. they could probably use the freedom

## Pre-Merge Checklist
dnm i mean to open this as a draft
- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

## Changelog
:cl:
balance: overhauls loadouts
/:cl:

### CODE REVIEW

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -20,6 +20,8 @@
 			continue
 		if (I.type in other_filter)
 			continue
+		if (special_type && (special_type = I.special_type)
```
Comment (Looreydev) [inline code/modules/uplink/uplink_items.dm:23]:
```suggestion
		if (special_type && (special_type = I.special_type))
```

**Comment by Looreydev** [comment]
Honestly, I like the idea of being able to pick a loadout yourself instead of only having a few static routes to go, obviously it can induce powergaming as we already talked about on Discord, just gotta find the right balance between the choice of items and how many points you give people, but that isn't done within a day.

File: code/modules/uplink/uplink_items.dm (hunk 1 above)
Comment (lolman360) [reply code/modules/uplink/uplink_items.dm:23]: intended

Comment (ScheveningenZonn) [reply code/modules/uplink/uplink_items.dm:23]: no you have to close parenthesis you DOOFUS

Comment (lolman360) [reply code/modules/uplink/uplink_items.dm:23]: i dont want this to be merged yet thus i make it not compule

**Comment by ScheveningenZonn** [comment]
flagging as DNM until you think its ready to review
